floodviz/static/js/hydrograph.js

```diff
@@ -286,17 +291,35 @@ var hydromodule = function (options) {
 	 * corresponding map site tooltip.
 	 */
 	self.series_tooltip_show = function (d) {
-		const padding = 3;
+		const padding = 4;
```

Comment:
Eventually I suspect we are going to want things like padding in a style sheet. This is fine for now.